src/Spryker/Zed/EventBehavior/Business/Model/TriggerManager.php

```diff
@@ -80,14 +90,20 @@ public function triggerRuntimeEvents()
 
         $processId = RequestIdentifier::getRequestId();
         try {
-            $events = $this->queryContainer->queryEntityChange($processId)->find()->getData();
-            static::$eventBehaviorTableExists = true;
-        } catch (PropelException | ConnectionException | EventBehaviorQueryNotExistsException $e) {
```

Comment:
`EventBehaviorQueryNotExistsException` no longer used, and can be deprecated.
1. `Throw` annotation in the query container should be deprecated. 
2. Functionality that throws an exception in query container should be moved to protected method and deprecated.
3. The exception class should be deprecated.